index.html

```diff
@@ -1,65 +1,49 @@
 <!DOCTYPE html>
 <html ng-app="oppiaGithubPages">
 
-  <head>
-    <meta charset="utf-8">
-    <base href="/">
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="Oppia : A tool for creating interactive tutors">
-
-    <link rel="stylesheet" type="text/css" media="screen" href="static/css/stylesheet.css">
-
-    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.25/angular.min.js"></script>
-    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.25/angular-route.js"></script>
-    <script src="app.js"></script>
-
-    <title ng-bind="$root.title"></title>
-  </head>
-
-  <body>
-    <div id="header_wrap" class="outer">
-      <header class="inner" style="max-width: 900px;">
-        <a id="forkme_banner" href="https://github.com/oppia/oppia">
-          View on GitHub
-        </a>
-        <h1 id="project_title">
-          <a href="https://www.oppia.org" class="anchor">Oppia</a>
-        </h1>
+<head>
```

Comment:
This should be indented by 2, since it's inside the <html> tag in the DOM tree (i.e., it isn't a sibling)